article/helpers.py

```diff
@@ -40,6 +43,19 @@ def persist_article(self, article_uuid):
     def retrieve_articles(self):
         return self.request.session.get(self.SESSION_KEY, [])
 
+    def article_read_count(self, category):
```

Comment:
this looks same as the method in the other class? Should be in `BaseArticleReadManager`

Reply:
I had this discussion with @richtier already.  Abstract classes cannot have concrete methods. Having a concrete base class would mean adding another level of inheritance, which seems unnecessary and a bit pointless when trying to save 3 lines of code. I'd rather turn the abstract class into a concrete class and move the common method there.